course_discovery/apps/api/v1/tests/test_views/test_programs.py

```diff
@@ -42,6 +43,21 @@ def setup(self, client, django_assert_num_queries, partner):
         self.partner = partner
         self.request = request
 
+    def _program_data(self):
+        course_run = CourseRunFactory()
+        organization = OrganizationFactory()
+        return {
+            "title": "Test Program",
+            "type": "XSeries",
+            "status": "active",
+            "marketing_slug": "edX-test-program",
+            "course_runs": [course_run.key],
+            "min_hours_effort_per_week": 10,
+            "max_hours_effort_per_week": 20,
+            "authoring_organizations": [organization.key],
```

Comment:
For all those many to many fields we should have multiple entries to verify things are working as expected with multiple entries. We should also verify update of program with multiple entries(intersection, union)